code/Modules/Props/Triggers/GarbageTrigger.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Sandbox;
+using ssl.Modules.Items.Carriables;
+
+namespace ssl.Modules.Props.Triggers
+{
+    [Library("ssl_garbage_trigger")]
+    [Hammer.AutoApplyMaterial( "materials/tools/toolstrigger.vmat" )]
+    [Hammer.Solid]
+    public partial class GarbageTrigger : BaseTrigger
+    {
+        public override void OnTouchStart(Entity toucher)
+        {
+            base.OnTouchStart(toucher);
+            toucher.Delete();
+        }
+
+        public override bool PassesTriggerFilters(Entity other)
+        {
+            return other.Tags.Has(Item.Tag);
+        }
+    }
+}
```

Comment:
question: i think i have see this before

Reply:
Lots of your PR are not based on develop, do we need to explore more options to improve the process ?

Reply:
I just forgot to checkout on develop when branching, everything is fixed now